src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_MenuFlyout.cs

```diff
@@ -69,6 +69,80 @@ public async Task When_Native_AppBarButton_And_Managed_Popups()
 			}
 		}
 
+		[TestMethod]
+		[RequiresFullWindow]
```

Comment:
Yes it's very likely the test is failing due to #9282

You can disable it on Mac:

```suggestion
		[RequiresFullWindow]
#if __MACOS__
		[Ignore("Currently fails on macOS, part of #9282 epic")]
#endif
```

Note: don't forget conventional commits when applying the suggestion from GitHub.

Reply:
> Note: don't forget conventional commits when applying the suggestion from GitHub.

I prefer amending my commit from my device to keep the PR history clean :)